src/components/rocket.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
+import { useDispatch } from 'react-redux';
+import { bookRocket, cancelBooking } from '../redux/rockets/rockets';
+
+const Rocket = ({ rocket }) => {
+  const dispatch = useDispatch();
+
+  const cancelBookedRocket = (id) => {
+    dispatch(cancelBooking(id));
+  };
+
+  const activateBooking = (id) => {
+    dispatch(bookRocket(id));
+  };
+  return (
+    <li key={rocket.id}>
+      <div className="flex justify-start gap-12">
+        <div className="min-w-[25rem]">
+          <img
+            src={rocket.flickr_images[0]}
+            className="object-cover h-[15rem] w-full rounded-sm"
+            alt="rocketname"
+          />
+        </div>
+        <div className="rocket-description">
+          <h2 className="text-[1.5rem] font-bold">{rocket.rocket_name}</h2>
+          <span>
+            {rocket.reserved ? (
+              <p className="text-[0.8rem] rounded-md p-1 text-white mt-2 bg-sky-500 w-fit">Reserved</p>
+            ) : null}
+          </span>
```

Comment:
-  Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
You can use the syntax:
`{ rocket.reserved && ( // code for reserved rocket)}
{ !rocket.reserved && ( // code for not reserved rocket)}`